src/ui/Services/CourseMapper.cs

```diff
@@ -44,16 +44,16 @@ public SearchAndCompare.Domain.Models.Course MapToSearchAndCompareCourse(ApiClie
                 Duration = MapCourseLength(courseEnrichmentModel.CourseLength),
                 Name = ucasCourseData.Name,
                 ProgrammeCode = ucasCourseData.CourseCode,
-                Provider = provider,
-                ProviderCodeName = ucasInstData.InstBig, // ???
+                Provider = provider,                
+                ProviderCodeName = ucasInstData.InstBig, // ???   
```

Comment:
added trailing whitespace, reformat the file

Reply:
whats `// ???`

Reply:
That's been in there since the file was first created. It doesn't mean anything so I'll remove it